extensions/mssql/src/main.ts

```diff
@@ -170,7 +170,7 @@ export async function activate(context: vscode.ExtensionContext): Promise<IExten
 	var disposable = provider.registerOnQueryComplete(params => {
 		console.log(params.ownerUri);
 		disposable.dispose();
-	});
+	}) as any as vscode.Disposable;
```

Comment:
Can you link to the failing build? We shouldn't need to do this...

Reply:
Oh wait hold on, oh no I checked in my test code 🤦 

I'm going to close this and open another PR to delete this...